pkg/planner/core/collect_column_stats_usage.go

```diff
@@ -39,18 +32,18 @@ const (
 // Histogram-needed columns are the columns whose histograms are utilized when making query plans, which usually occur in the conditions pushed down to DataSource.
 // The set of histogram-needed columns is the subset of that of predicate columns.
 type columnStatsUsageCollector struct {
-	// collectMode indicates whether to collect predicate columns and/or histogram-needed columns
-	collectMode uint64
```

Comment:
I think we can remove `collectHistNeededColumns` now. And remove the relative comment from other fields as well.